### DISCUSSION TITLE
LastActiveLimitNotReached Tag

### DISCUSSION THREAD

**Discussion by oOStroudyOo** [StuffAnThings/qbit_manage#459] [Q&A]
Hello,

I'm using the `last_active` variable on the share limits feature which works perfectly, however I'd quite like torrents to **not** be tagged as LastActiveLimitNotReached.

Is this even possible?

Thank you.

**Comment by AN1MATEK** [comment]
I'm also not a big fan of extra tags. I think it would be cleaner if it would just read the values returned by the API and act on them. Theoretically would also be the case for the `MinSeedsNotMet` and `MinSeedTimeNotReached` but hey, it works fine and it's free software so no biggie.

And yes, it is possible. I used to use `autoremove-torrents`, which did some of this stuff quite well and did not need Tags. But when `qbit_manage` added this functionality I diched the former.

Off topic: I don't think this "Discussion" section is very relevant... If you think this is an issue go ahead and open one, or if you want to actually discuss it would be better in Discord where everybody is quite active. Just my 2 cents.
